src/components/WorkOrder/Photos/hooks/useFileUpload.js

```diff
@@ -27,6 +27,7 @@ const useFileUpload = (workOrderReference, onSuccess) => {
     const uploadResult = await uploadFiles(
       files,
       workOrderReference,
+      '',
```

Comment:
Just confirming: the `uploadGroupLabel` will get set elsewhere after the upload?